.github/workflows/assigner.yml

```
@@ -62,7 +63,7 @@ jobs:
         FLAGS+=" -r ${{ github.event.repository.name }}"
         FLAGS+=" -M MAINTAINERS.yml"
         if [ "${{ github.event_name }}" = "pull_request_target" ]; then
-          FLAGS+=" -P ${{ github.event.pull_request.number }} --updated-manifest pr_west.yml"
+          FLAGS+=" -P ${{ github.event.pull_request.number }} --updated-manifest pr_west.yml" --updated-mantainer-file pr_MAINTAINERS.yml"
```

Probably meant to be:
```suggestion
          FLAGS+=" -P ${{ github.event.pull_request.number }} --updated-manifest pr_west.yml --updated-mantainer-file pr_MAINTAINERS.yml"
```
?